### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by xmulligan**
For people that want to contribute or suggest content for ebpf.io, it would be great if we could have some issue templates for them so they know where to get started.

### CODE REVIEW

**Review by qmonnet — APPROVED** [review]
Looks good, apart from the inconsistent title case on section titles :see_no_evil:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+### Page(s) Affected
+<!--
+Please list the page or pages this bug is present on. Please file one issue
+for each bug you find. Only file an issue for a multi-page bug if the bug is
+indeed the *same* bug, across the pages.
+-->
+
+### What needs to be fixed?
```
Comment (xmulligan) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
### What Needs To Be Fixed?
```

File: .github/ISSUE_TEMPLATE/content_bug_report.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: Incorrect Content
+about: Found a typo? Is content out of date? Need a clarification?
+---
+
+### What needs to be fixed?
```
Comment (xmulligan) [inline .github/ISSUE_TEMPLATE/content_bug_report.md:6]:
```suggestion
### What Needs To Be Fixed?
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+One paragraph explanation of the design you'd like to see added.
+-->
+
+### Motivation
+<!--
+Why are we doing this? What use cases does it support? What is the
+expected outcome?
+-->
+
+### Rationale and alternatives
```
Comment (xmulligan) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]:
```suggestion
### Rationale and Alternatives
```

**Review by qmonnet — APPROVED** [review]

**Review by borkmann — APPROVED** [review]
:100:

**Pull request merged**